app/WebParsing/ReqParser.hs

```diff
@@ -26,6 +26,32 @@ completionPrefix = Parsec.choice (map (Parsec.try . Parsec.string) [
     ])
     >> Parsec.spaces
 
+programPrefix :: Parser ()
+programPrefix = Parsec.choice (map caseInsensitiveStr [
+    "admission to",
+    "enrolment in the ",
+    "enrolment in an ",
+    "enrolment in a ", -- trailing space to prevent matching "enrolment in A..."
+    "enrolment in"
+    ])
+    >> Parsec.spaces
+
+degreeType :: Parser String
+degreeType = do
+    Parsec.spaces
+    degree <- Parsec.choice $ map caseInsensitiveStr [
+        "major",
+        "specialist"
```

Comment:
Make sure to add `minor` here as well, and tests for it.

Reply:
Minor doesn't appear anywhere in the timetable. Should we still add it?

I think that it is not by luck that we don't have "minor"s in the timetable, because every requirement that requires a minor would also accept a major/specialist, so the timetable would just list them as "program". For example, instead of saying "Enrolment in the CS specialist, major, or minor", the timetable would just say "Enrolment in the CS program". So it seems like "minor" will never appear?

I can still add it of course. Just double checking if we should add it :)

Reply:
Yes you should add it, it's good for future-proofing. In principle there could be courses that are accessible only to students in a minor program; we must be careful not to overfit to the existing calendar.